tests/cnf/core/network/metallb/tests/bgp-remote-as-dynamic.go

```diff
@@ -119,6 +121,74 @@ var _ = Describe("BGP remote-dynamicAS", Ordered, Label(tsparams.LabelDynamicRem
 						fmt.Sprintf("The remoteASN does not match the expected AS: %d", 0))
 				})
 		})
+
+		Context("multi hop", func() {
+			var (
+				externalAdvertisedIPv4Routes = []string{"192.168.100.0/24", "192.168.200.0/24"}
+				externalAdvertisedIPv6Routes = []string{"2001:100::0/64", "2001:200::0/64"}
```

Comment:
Please move externalAdvertisedIPv4Routes  and externalAdvertisedIPv6Routes to the upper context as well.

Reply:
Of course.
Moved